src/app/pusher.service.ts

```diff
@@ -12,7 +12,7 @@ export class PusherService {
   }
 
   initializePusher(): void {
-    this.pusher = new Pusher(environment.pusher.key, { authEndpoint: '/pusher/auth' });
+    this.pusher = new Pusher(environment.pusher.key, { authEndpoint: 'http://localhost:8989/pusher/auth' });
```

Comment:
Why was this added back? You should avoid hardcoding in your domain as much as possible.